Pacifique78/MyDiary
# #169542004 start using app versioning

#### What does this PR do?
This PR adds logics for app versioning

#### Description of Task to be completed
>- create two folders in server named V1 and V2
>- put codes of v1, v2 into V1, V2 folders respectfully
>- change the server to use both routes
>- change tests script in package.json to run both the tests
#### How should this be manually tested?
>- clone the repo
>- run `npm install` and `npm run dev` to start the server
>- use either postman or any other testing tool to see if all endpoints for both versions are working.
>- run `npm test` to see if all the test are passing
>- Check if the Travis test are passing

#### What are the relevant pivotal tracker stories?
#169542004

#### Screenshots (if appropriate)
![Screenshot (121)](https://user-images.githubusercontent.com/50268181/68134396-bc0a1f00-ff2a-11e9-8112-37994f88e5fa.png)
![Screenshot (123)](https://user-images.githubusercontent.com/50268181/68134399-bc0a1f00-ff2a-11e9-997c-74d4e3746083.png)
